richtext-commonmark/src/commonMain/kotlin/com/halilibo/richtext/commonmark/MarkdownParseOptions.kt

```diff
@@ -1,16 +1,32 @@
 package com.halilibo.richtext.commonmark
 
+import org.commonmark.Extension
+import org.commonmark.ext.autolink.AutolinkExtension
+import org.commonmark.ext.gfm.strikethrough.StrikethroughExtension
+import org.commonmark.ext.gfm.tables.TablesExtension
+
 /**
  * Allows configuration of the Markdown parser
  *
  * @param autolink Detect plain text links and turn them into Markdown links.
  */
-public data class MarkdownParseOptions(
-  val autolink: Boolean
-) {
+public data class MarkdownParseOptions(val extensions: List<Extension>) {
```

Comment:
since `richtext-commonmark` and `richtext-markdown` are now separate modules I'm not as opposed to exposing commonmark details as I had been before, but I still would prefer if extensions were exposed as a value class. 

```
value class MarkdownExtension private constructor(internal val value: Int) {
  companion object {
    val Tables = ...
    val AutoLink = ...
    ...
  }
}
```